### PR TITLE
Decode HTML entities in the documentation file (parameterData.json)

### PR DESCRIPTION
**Pull request by yukienomiya**
**Changes:**
Modified the renderItemDescriptionsAsMarkdown() function in the preprocessor.js so the HTML entities (like "&amp;" for "&") in the class/method/field descriptions are decoded. I used the node-html-entities library to decode the text.

The documentation file (data.json in the p5.js-website repository) is used to create the Reference translation files, so I'm trying to make it as readable as possible.

### CODE REVIEW

**Comment by limzykenneth** [comment]
Looks good, just one thing, can the `html-entities` dependency be moved to dev dependencies?

On another note, I've been looking into potential replacement for yuidocs and am trying it out in one of my projects. It is a considerable change but if it works out on my project, it may be viable to be explored for p5.js as well.

File: package.json [hunk 1]
```diff
@@ -145,7 +145,9 @@
     "not dead"
   ],
   "author": "",
-  "dependencies": {},
+  "dependencies": {
+    "html-entities": "^1.3.1"
```
Comment (outofambit) [inline package.json:149]: @yukienomiya could we move this to `devDevependencies` please?
(thanks @limzykenneth for noticing this!)

**Comment by outofambit** [comment]
> On another note, I've been looking into potential replacement for yuidocs and am trying it out in one of my projects.

@limzykenneth exciting! out of curiosity, what's the tool called?

**Comment by limzykenneth** [comment]
@outofambit It's [comment-parser](https://github.com/syavorsky/comment-parser), a JSDoc/YUIDoc style comment parser that just outputs the various comment blocks and the associated tags in the block as a javascript object. With a bit of extra work, we could potentially parse the output to fit what we need and repalce `data.json` with it.

File: package.json (hunk 1 above)
Comment (yukienomiya) [reply package.json]: Done! Thanks @limzykenneth :)

**Pull request merged**